packages/p2p/src/components/modal-manager/modals/create-ad-modal/create-ad-modal.scss

```diff
@@ -0,0 +1,16 @@
+.dc-modal__container_create-ad-modal {
```

Comment:
I believe you can add className to the Modal.Body component so you don't have to directly target it using this way